### DISCUSSION TITLE
Align/float image left, right or center

### DISCUSSION THREAD

**Discussion by foxtrotcz** [craftcms/ckeditor#132] [Ideas]
Hi,
it was option in Redactor to set image to float left or right of text.
![image](https://github.com/craftcms/ckeditor/assets/10333810/c32ad1eb-adf6-42e4-9add-9c163b8a8700)

Please add this option to CKEditor.
Thanks.

**Comment by SETU-WEB** [comment]
Agree on this one

**Comment by i-just** [comment]
Hi, I believe this might help you achieve what you’re after: https://github.com/craftcms/ckeditor/discussions/243#discussioncomment-9750506.
